schema/readout-FakeCardReader-make.jsonnet

```diff
@@ -4,12 +4,16 @@
     queue: "fake-elink"
 
     // Make a conf object for cardreader
-    conf(lid=0, inplim=10485100, rate=166, rawtype="wib", datfile="/tmp/frames.bin", qtms=2000) :: {
+    conf(lid=0, inplim=10485100, rate=166, rawtype="wib", datfile="/tmp/frames.bin", qtms=2000,
+         raw_win_tp=true, tp_datfile="/tmp/tp_frames.bin", tp_rate=66) :: {
```

Comment:
Typo? "raw_win_tp"?
Did you test the schema generation?

Reply:
Yes, I ran schema generation, as 
moo -M "`pwd`/../../appfwk/schema" compile fakereadout-app.jsonnet > ../test/tpenabled-fakereadout-commands.json
without this command producing errors. Perhaps this should be investigated.